src/source/reference/delete-campaign.rst

```diff
@@ -0,0 +1,24 @@
+.. index:: DELETE /campaign/{id}
+.. _delete_campaign:
+
+DELETE /campaign/{id}
+===============================
+
+It is possible to remove a campaign in one go. This does not remove the ads from your inventory; it only takes them offline for that particular campaign.
+
+.. _delete-campaign:
+
+.. list-table::
+ :widths: 20 80
+
+ * - Scope
+   - ``api_rw`` or ``console_rw``
+
+ * - Content-Type
+   - ``application/sellside.campaign-v2+json; charset=utf-8``
+
+Removes a campaign comepletely. If successful, the server responds with `HTTP 204`.
```

Comment:
remove from DB or set status to DELETED? best be clear, especially when we don't allow a second campaign with same vendorID.
For the time that we only allow 1 campaign per user this call might be discouraged (unless we limit to 1 active campaign per user)